jina/parsers/orchestrate/runtimes/container.py

```diff
@@ -61,5 +62,5 @@ def mixin_container_runtime_parser(parser):
         '--disable-auto-volume',
         action='store_true',
         default=False,
-        help='Do not automatically mount a volume for dockerized Executors.',
+        help=f'Do not automatically mount a volume for dockerized {pod_type.title()}s.',
```

Comment:
how do that play we the automatic docstring ?

Reply:
are you referring to the auto generated docstring and parameters that jina bot adds ?
If so, the bot actually runs the parser code and returns results

Reply:
okay perfect then